src/app/public/modules/data-manager/models/data-view-config.ts

```diff
@@ -56,6 +62,11 @@ export interface SkyDataViewConfig {
    * Indicates whether to display the search box for this view.
    */
   searchEnabled?: boolean;
+  /**
+   * Specifies placeholder text to display in the search input until users enter search criteria.
+   * @default 'Find in this list'
```

Comment:
ok, the interface can't actually have a default, I was just trying to communicate that is was the search text will be if they don't set anything, since that is the default in the search component

Reply:
In that case, should we add a sentence to the end of the description that says something like `If no value is provided, the search input displays the default "Find in this list" placeholder text.`? Adding a suggestion for this...
```suggestion
   * If no value is provided, the search input displays the default "Find in this list" placeholder text.
```